Utils/azuretools-core/src/com/microsoft/azuretools/utils/WebAppUtils.java

```diff
@@ -385,17 +386,31 @@ public static boolean isJavaWebApp(@NotNull WebApp webApp) {
      * We return a combined and refactored information as Java Runtime.
      */
     public static String getJavaRuntime(@NotNull final WebApp webApp) {
-        final String delimiter = "  ";
+        String webContainer;
         switch (webApp.operatingSystem()) {
             case WINDOWS:
-                final String refactoredJavaVersion = "Java" + webApp.javaVersion().toString();
-                return String.join(delimiter, webApp.javaContainer(), webApp.javaContainerVersion(), refactoredJavaVersion);
+                webContainer = webApp.javaContainer() == null ? null : webApp.javaContainer().toLowerCase();
+                return String.format("%s %s (Java%s)",
+                    StringUtils.capitalize(webContainer), webApp.javaContainerVersion(), webApp.javaVersion().toString());
```

Comment:
`StringUtils.capitalize(null)  = null`, is this what you expect if `webApp.javaContainer() == null` ?

Reply:
Yes, if any part of the format is invalid, we show it as it is, like what we do for Linux runtime.